charts/prometheus-elasticsearch-exporter/templates/deployment.yaml

```diff
@@ -131,7 +131,7 @@ spec:
                     {{- range .Values.extraArgs }}
                     {{ . | quote }},
                     {{- end }}
-                    "--web.listen-address=:{{ .Values.service.httpPort }}",
+                    "--web.listen-address=:{{ .Values.deployment.httpPort }}",
```

Comment:
hi, @zeritti thank you for the review!
sure I will add a note that I add `.deployment.httpPort` and `service.httpPort` is no longer bound necessarily to the deployment port, however, I might not fully understand you regarding why this change is considered breaking changes, if the user upgrade to this version of the chart it will get default value that will work for him even the user customize the `service.httpPort` since we use named port anyway